packages/scene-composer/src/components/panels/scene-settings/MatterportIntegration.tsx

```diff
@@ -64,15 +66,27 @@ export const MatterportIntegration: React.FC = () => {
     return () => unsubscribe(onUpdated);
   }, []);
 
-  const onConnectionNameChange = useCallback(async (event) => {
+  const enableMatterportViewer = useMemo(() => {
```

Comment:
`useMemo` is not necessary for simple operations and adds complexity. It's completely fine to just rune this computation on every render and won't have any impact on performance. should avoid useMemo unless its needed

Reply:
Removed useMemo